pages/index.vue

```diff
@@ -25,19 +16,23 @@
                 :enabled="true" />
         </b-row>
 
-
-        <!-- Data dictionary file loading area -->
+        <!-- Shows file contents -->
         <b-row>
-            <h3>{{ uiText.dataDictionaryHeader }}</h3>
+            <b-table
+                v-if="dataTable.length > 0"
+                outlined
+                sticky-header
+                :items="dataTable"
+                data-cy="data-table-display" />
+            <div v-else>
+                Please provide a data table to see a preview here.
```

Comment:
:cherries: I think it'd be visually better if we had this text styled like the instruction text in the categorization page. I believe the css classs for it is `instructions-text`.